tsconfig.base.json

```diff
@@ -35,6 +35,8 @@
       "@fxa/shared/account/account": [
         "libs/shared/account/account/src/index.ts"
       ],
+      "@fxa/shared/assets/*": ["libs/shared/assets/src/lib/*"],
```

Comment:
Is there a reason this references a wildcard rather than a specific entrypoint barrel file?

Reply:
Sounds good, thanks for the explanation!